src/aws-elasticache/serverless-cache.ts

```diff
@@ -0,0 +1,531 @@
+import {
+  IResource,
+  Lazy,
+  Names,
+  Resource,
+  Stack,
+  aws_elasticache,
+  aws_ec2,
+  Token,
+  aws_iam,
+  ArnFormat,
+} from 'aws-cdk-lib';
+import { Metric, MetricOptions } from 'aws-cdk-lib/aws-cloudwatch';
+import { IKey } from 'aws-cdk-lib/aws-kms';
+import { Construct } from 'constructs';
+import { DailySnapshotTime } from './daily-snapshot-time';
+import { IUserGroup } from './user-group';
+import { Engine } from './util';
+
+/**
+ * An ElastiCache Serverless Cache
```

Comment:
Let's keep it in line with IIamUser and others, just in case.

```suggestion
 * Interface for an ElastiCache Serverless Cache
```